cassandra/1.cql

```diff
@@ -1,13 +1,13 @@
 CREATE KEYSPACE devgym
   WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 };
-
+/
```

Comment:
It may not work via cmd anymore. Since we are going to use DataLoader, we do not need to run this script manually anymore.